nemo_curator/modules/fuzzy_dedup.py

```diff
@@ -146,8 +201,19 @@ def minhash64(
         """
         if not isinstance(ser, cudf.Series):
             raise TypeError("Expected data of type cudf.Series")
-        seeds = cudf.Series(seeds, dtype="uint64")
-        return ser.str.minhash64(seeds=seeds, width=char_ngram)
+        if not MINHASH_PERMUTED_AVAILABLE:
+            self._logger.warning(
+                "Using an older implementation of minhash, update to cudf >= 24.12"
```

Comment:
```suggestion
                "Using an outdated minhash implementation, please update to cuDF version 24.12 or later for improved performance."
```

Reply:
Nit: Thinking about this a bit more, not sure if it's worth calling outdated if cuDF 24.12 hasn't even been released yet. I wonder if we could reword to share instructions on how users can install `nemo_curator[_nightly]` variants to get a faster variant of minhashes rather than asking them to install cuDF which folks might not be familiar with.